source/table.h

```diff
@@ -106,6 +109,7 @@ struct spv_context_t {
   const spv_opcode_table opcode_table;
   const spv_operand_table operand_table;
   const spv_ext_inst_table ext_inst_table;
+  bool permissive;
```

Comment:
Need a better comment on this.
Permissive here means: binary parsing must still succeed (and hence we must have a valid instruction table for it), but then no validation rules are checked.